tests/test_yaml_metaworkflow.py

```diff
@@ -34,9 +35,9 @@ def test_metaworkflow():
               "value_type": "json"
             }
           ],
-          "institution": "/institutions/hms-dbmi/",
+          "submission_centers": ["hms-dbmi", "smaht-dbmi"],
```

Comment:
For submission centers we should agree on this with Doug but I was using `ParkLab` or probably better yet `smaht-dac`